HyprMXAdapter/src/main/java/com/chartboost/mediation/hyprmxadapter/HyprMXAdapter.kt

```diff
@@ -770,4 +777,18 @@ class HyprMXAdapter : PartnerAdapter {
         HyprMXErrors.AD_SIZE_NOT_SET -> ChartboostMediationError.CM_LOAD_FAILURE_INVALID_BANNER_SIZE
         else -> ChartboostMediationError.CM_PARTNER_ERROR
     }
+
+    /**
+     * Checks that the HyprMX initialization state has completed. If so, then run the function;
+     * otherwise, do nothing.
+     *
+     * @param function the function that will be run after the initialization state check was successful.
+     */
+    private fun checkHyprMxInitStateAndRun(function: () -> Unit) {
+        if (HyprMX.getInitializationState() != HyprMXState.INITIALIZATION_COMPLETE) {
+            PartnerLogController.log(CUSTOM, "Ignoring $function until HyprMX SDK has initialized.")
```

Comment:
"Ignoring until" isn't the right message. There's no "until" in this code—it's a one-time check and if the conditional is not met the action doesn't take place. Using "until" conveys we're polling for the initialized state and then re-running the passed in function.

Reply:
How does this sound?

> "Cannot run $function. The HyprMX SDK has not initialized."

Reply:
I think that's fine.